src/Data/Food/Process.elm

```diff
@@ -199,16 +200,16 @@ decodeProcess impactsDecoder =
 encode : Process -> Encode.Value
 encode process =
     Encode.object
-        [ ( "category", encodeCategory process.category )
-        , ( "comment", EncodeExtra.maybe Encode.string process.comment )
+        [ ( "name", Encode.string (nameToString process.name) )
         , ( "displayName", EncodeExtra.maybe Encode.string process.displayName )
-        , ( "id", Encode.string process.id_ )
-        , ( "identifier", encodeIdentifier process.identifier )
         , ( "impacts", Impact.encode process.impacts )
-        , ( "name", Encode.string (nameToString process.name) )
-        , ( "source", Encode.string process.source )
-        , ( "system_description", Encode.string process.systemDescription )
         , ( "unit", encodeStringUnit process.unit )
+        , ( "identifier", encodeIdentifier process.identifier )
+        , ( "categories", Encode.list encodeCategory process.categories )
+        , ( "system_description", Encode.string process.systemDescription )
+        , ( "comment", EncodeExtra.maybe Encode.string process.comment )
+        , ( "id", Encode.string process.id_ )
+        , ( "source", Encode.string process.source )
```

Comment:
Could we keep the orders of the keys here as everything is sorted alphabetically on the Elm side?